### PR TITLE
update further reading to include A rollup-centric ethereum roadmap

### PR DESCRIPTION
**Pull request by MiConnell**
## Description
This PR adds a link to A rollup-centric ethereum roadmap, which is mentioned at the end of the video from the `More of a visual learner` section

### CODE REVIEW

**Review by wackerow — APPROVED** [review]
Thanks @MiConnell! Just noting we do have this article linked on the `/en/eth2/shard-chain` shard chains page, but agree it provides value on this page as well.

File: src/content/developers/docs/scaling/index.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ _Note the explanation in the video uses the term "Layer 2" to refer to all off-c
 
 ## Further reading {#further-reading}
 
+- [A rollup-centric ethereum roadmap by Vitalik Buterin](https://ethereum-magicians.org/t/a-rollup-centric-ethereum-roadmap/4698)
```
Comment (minimalsm) [inline src/content/developers/docs/scaling/index.md:86]:
```suggestion
- [A rollup-centric ethereum roadmap](https://ethereum-magicians.org/t/a-rollup-centric-ethereum-roadmap/4698) _Vitalik Buterin_
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Thanks for the PR @MiConnell 💪 Just need a small tweak to enforce the convention we use when referring to article authors.

**Comment by minimalsm** [comment]
[ ] - Note to self - add this to style guide

**Comment by minimalsm** [comment]
Thanks for the changes! (I've added hopefully clearer guidance around this in #4773). Merging 🚀 Don't forget to claim [claim your POAP](https://ethereum.org/en/contributing/#poap) for 2021 if you haven't already 🥳!

**Comment by MiConnell** [comment]
awesome, thank you!

**Pull request merged**